Sources/DataPipeline/DataPipelineConfigOptions.swift

```diff
@@ -35,6 +30,9 @@ public struct DataPipelineConfigOptions {
         self.writeKey = writeKey
     }
 
+    /// Site Id that will be used for tracking data migration
+    public var siteId: String = ""
```

Comment:
Can we avoid storing `siteId` here? We recently removed it from `SdkConfig`, if there is an option to avoid storing it, I would prefer not storing it here again.

Reply:
We need `siteId` for migration specifically to fetch the `profileIdentifier` and in `fileStorage`. We would not be using `siteId` directly through config but would be iterating down to the methods as and where required. Since we discussed the same 1:1 on slack and now that we both are on same page, I am resolving this comment.